utils/decision.py

```diff
@@ -1,60 +1,51 @@
-from enum import Enum, auto
+from abc import ABC, abstractmethod
 
-from utils.result import ResultTypes
+from utils.result import ResultType
 
 
-class DecisionStrategies(Enum):
-    Strict = auto()
-    Majority = auto()
-    Unanimous = auto()
+class DecisionStrategy(ABC):
+    @staticmethod
```

Comment:
why is this one static but the other abc's things arent?

Reply:
What do you mean by other abc's stuff? It is static since it is not dependent on the instance of the DecisionStrategy

Reply:
The other abstract classes from this PR, or are there not multiple? Don't they have the same property?

Reply:
![image](https://github.com/zerohour-phishing-detection/zpd-server/assets/30039677/0ba4645f-e15a-4c6e-9cc8-92f3a47a2700)

This is the other abstract class that exists. I am not sure if this should also be abstract.

Reply:
Yea fair enough, that one may have some use to instances